src/style/buttons.scss

```diff
@@ -1,17 +1,21 @@
+@import './variables.scss';
+
 .btn {
```

Comment:
Universal button styling to make links look a react like buttons still has some minor bugs need to eliminate any stylings not needed